headquarter/argocd/application.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   project: headquarter
   source:
     repoURL: https://github.com/RHTE-2023-Edge-Lab/rhte-gitops.git
-    targetRevision: main
+    targetRevision: slallemand/reorg-warehouse-manifests
```

Comment:
Once merged, the target revision should be "main", right ?

Reply:
right !

Reply:
also working on resolving conflicts ^^